_data/members.yml

```diff
@@ -31,16 +31,6 @@
   interests: Open Source Software, Accessibility, Inclusion, Community
   interets: Logiciels libres, Accessibilité, Inclusion, Communauté
   picture_path: assets/images/profile-guillaume.png
-- name: My Thuy Pham-Vo
```

Comment:
Sorry to see My go, but this change makes sense at this point. Keeping in history for when she comes back to the team ;)